<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RelationWindow</class>
 <widget class="QDialog" name="RelationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTableView" name="relatedTable">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>221</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="nRelatedTable">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>221</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteRelation">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-------&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addRelation">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-------</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filterRelated">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Filter related...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filterNRelated">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Filter unrelated...</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameOfEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name goes here</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RelationWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RelationWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
